kernel/src/expressions/scalars.rs

```diff
@@ -472,6 +472,7 @@ impl PrimitiveType {
 
 #[cfg(test)]
 mod tests {
+    use core::panic;
```

Comment:
do we need this?

Reply:
Ah this was from the test I'd tried to write for timestamp. Weird that rust didn't detect this :/